crates/sui-indexer/src/indexer_reader.rs

```diff
@@ -833,9 +764,14 @@ impl IndexerReader {
                     "InputObject and OutputObject filters are not supported, please use AffectedObject instead.".into()
                 ))
             }
+            Some(TransactionFilter::ToAddress(_)) => {
+                return Err(IndexerError::NotSupportedError(
+                    "ToAddress filter is not supported, please use FromOrToAddress instead.".into()
```

Comment:
@amnn `FromOrToAddress` filter is not supported by RPC API atm https://github.com/MystenLabs/sui/blob/95792043bb29cd1b42a4596f3d238f6353329a76/crates/sui-json-rpc-types/src/sui_transaction.rs#L2432
So with `ToAddress` being removed there are no ways to query it in RPC.

Reply:
@amnn got it. Thank you very much.